tests/Unit/RmStepTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+use WordPress\Blueprints\BlueprintException;
+use WordPress\Blueprints\Model\DataClass\RmStep;
+use WordPress\Blueprints\Runner\Step\RmStepRunner;
+use WordPress\Blueprints\Runtime\NativePHPRuntime;
+
+
+beforeEach(function() {
+    $this->step = new RmStepRunner();
+    $documentRoot = sys_get_temp_dir();
+    $this->runtime = new NativePHPRuntime($documentRoot);
+    $this->step->setRuntime($this->runtime);
+});
+
+it('should remove a directory when an absolute path provided', function() {
+    $unresolvedDirToRemove = "dir";
+    $resolvedDirToRemove = $this->runtime->resolvePath($unresolvedDirToRemove);
+    mkdir($resolvedDirToRemove, 0777, true);
+    expect(file_exists($resolvedDirToRemove))->toBeTrue();
+
+    $input = new RmStep();
+    $input->path = $resolvedDirToRemove;
+
+    $this->step->run($input);
+
+    expect(file_exists($resolvedDirToRemove))->toBeFalse();
+});
+
+it('should remove a directory when a relative path provided', function () {
+    $unresolvedDirToRemove = "dir";
+    $resolvedDirToRemove = $this->runtime->resolvePath($unresolvedDirToRemove);
+    mkdir($resolvedDirToRemove, 0777, true);
+    expect(file_exists($resolvedDirToRemove))->toBeTrue();
+
+    $input = new RmStep();
+    $input->path = $unresolvedDirToRemove;
+
+    $this->step->run($input);
+
+    expect(file_exists($resolvedDirToRemove))->toBeFalse();
+});
+
+it ('should remove a directory with a subdirectory', function () {
+    $unresolvedSubDir = "dir/subdir";
+    $resolvedSubDir = $this->runtime->resolvePath($unresolvedSubDir);
+    mkdir($resolvedSubDir, 0777, true);
+    expect(file_exists($resolvedSubDir))->toBeTrue();
```

Comment:
This test could use `new FileSystem()` instance from Symfony and then it wouldn't need to check if `mkdir` failed since FileSystem throws an exception for us.

Reply:
I added the assertion before the tested code execution for robustness. Although not entirely impossible, I doubt it could actually fail. So it mainly was a mean to highlight the change. Like: "see, it's there. now, we do sth. and it's gone. tada."

But I agree, this API is well thought out, and using it here can be beneficial.

Reply:
> Although not entirely impossible, I doubt it could actually fail

It's more common than you'd think – read-only filesystems, already existing directories, invalid paths and other corner cases happen all the time.